ci/travis-build.sh

```diff
@@ -4,6 +4,10 @@ if [ -n "${CONDA}" ]; then
     conda build ci/conda
     mkdir -p dist
     cp -av /home/travis/miniconda/conda-bld/*/*.tar.bz2 dist/
+elif [ -n "${PIP}" ]; then
```

Comment:
Using `PIP` to trigger cibuildwheel is a bit funky because it isn't obvious that `PIP` would do that (unless cibuildwheel is universally being used, which it isn't yet).

I think this wants to be a `BUILDMODE` variable or something like that, where the values can be `CONDA`, `CIBUILDWHEEL`, etc.

Reply:
Done